tox.ini

```diff
@@ -1,6 +1,13 @@
 [tox]
-envlist = py27, py33, py34, py35, py36
+envlist = {py27,py33,py34,py35,py36}-{pytest,behave}
```

Comment:
`cd test && behave` doesn't seem to work:
```
py35-behave runtests: commands[0] | cd test && behave
WARNING:test command found but not installed in testenv
  cmd: /usr/bin/cd
  env: /home/meeuw/git/mycli/.tox/py35-behave
```
But this isn't necessary with your setup.cfg fixes, let's wait until that's merged.